typha/pkg/config/config_params.go

```diff
@@ -115,12 +115,12 @@ type Config struct {
 	SnapshotCacheMaxBatchSize int `config:"int(1,);100"`
 
 	ServerMaxMessageSize                 int           `config:"int(1,);100"`
-	ServerMaxFallBehindSecs              time.Duration `config:"seconds;300"`
+	ServerMaxFallBehindSecs              time.Duration `config:"seconds;300"` //nolint:staticcheck // Ignore ST1011 don't use unit-specific suffix
```

Comment:
@fasaxc Can we remove the `Secs` part in these config items? Will that break existing configurations?